march_hardware_interface/src/march_hardware_interface.cpp

```diff
@@ -210,7 +210,7 @@ void MarchHardwareInterface::read(const ros::Time& /* time */, const ros::Durati
 
     // Apply exponential smoothing to velocity obtained from encoder with
     // alpha=0.2
-    joint_velocity_[i] = filters::exponentialSmoothing(joint_velocity, joint_velocity_[i], 0.2);
+    joint_velocity_[i] = 0.2 * joint_velocity + 0.8 * joint_velocity_[i];
```

Comment:
It think it's better to use a constant here declared in `MarchHardwareInterface` something like
```c++
public:
...
static constexpr double ALPHA = 0.2;
...
```
Then change this line to
```suggestion
    joint_velocity_[i] = MarchHardwareInterface::ALPHA * joint_velocity + (1 - MarchHardwareInterface:ALPHA) * joint_velocity_[i];
```